go/hyperkit.go

```diff
@@ -367,32 +375,49 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	a = append(a, "-m", fmt.Sprintf("%dM", h.Memory))
 
 	a = append(a, "-s", "0:0,hostbridge")
+	a = append(a, "-s", "31,lpc")
+
+	nextSlot := 1
```

Comment:
we don't support multiple disks in the API, not multiple other devices, but I see 9p mounts is an array, but then it's not exposed via the sample CLI. Do you plan to support multiple 9p sockets via `linuxkit run`?

To be clear, I don't have strong objections to making this dynamic.